docs/make.jl

```diff
@@ -0,0 +1,32 @@
+using Documenter, StatsBase
+
+makedocs(
+    format = :html,
+    sitename = "StatsBase.jl",
+    # options
+    modules = [StatsBase],
+    pages = ["index.md",
+             "weights.md",
+             "means.md",
+             "scalarstats.md",
+             "robust.md",
+             "deviation.md",
+             "cov.md",
+             "counts.md",
+             "ranking.md",
+             "sampling.md",
+             "empirical.md",
+             "signalcorr.md",
+             "misc.md",
+             "statmodels.md"]   
+)
+
+deploydocs(
+    # options
+    repo = "github.com/JuliaStats/StatsBase.jl.git",
+    target = "build",
+    julia  = "nightly",
```

Comment:
The problem is that `0.6` will be outdated when 1.0 comes out, and we'll likely forget to update that file. We need a better solution (like `release`), but for now I'd rather use `nightly`.